rpm/glacier-music.spec

```diff
@@ -34,13 +35,15 @@ Music application for nemo mobile
 %setup -q -n %{name}-%{version}
 
 %build
-%qtc_qmake5  VERSION=%{version}
-
-make %{?_smp_mflags}
+cmake -B build \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DCMAKE_INSTALL_PREFIX=%{_prefix} \
+	-DCMAKE_INSTALL_LIBDIR=%{_lib}
+make -C build VERBOSE=1
```

Comment:
I would recommend:
1. Using CMake for all three (configure, build, install) to free the spec from the hardcoded make commands.
2. Use `CMAKE_VERBOSE_MAKEFILE=ON` in configure instead of VERBOSE=1 on build step.
3. Add the source dir to the configure command and keep it in the last line.

```
%build
cmake -B build \
	-DCMAKE_VERBOSE_MAKEFILE=ON \
	-DCMAKE_BUILD_TYPE=Release \
	-DCMAKE_INSTALL_PREFIX=%{_prefix} \
	-DCMAKE_INSTALL_LIBDIR=%{_lib} \
	.
cmake --build build

%install
rm -rf %{buildroot}
DESTDIR=$RPM_BUILD_ROOT cmake --build build --target install
```

To further improve the portability and system integration you can also try to use `%cmake` macro ([Mer](https://git.sailfishos.org/mer-core/cmake/blob/master/rpm/macros.cmake#L8), [Fedora](https://src.fedoraproject.org/rpms/cmake/blob/master/f/macros.cmake#_16), [openSUSE](https://build.opensuse.org/package/view_file/openSUSE:Factory/cmake/cmake.macros?expand=1)), but I would not recommend that yet, because it is not done right in Mer, nor Fedora (wow, openSUSE actually did it :+1: right).
Like the macros for qmake, there should be `%cmake` for the configure step, `%cmake_build` for build and `%cmake_install` for install.
The configure macro already takes care of the prefix, libdir, and some other variables, but it also should provide a macro without all those `-D` arguments to be able to customize any step of the build from a single place.

I would suggest to maybe use `%cmake` for configure and plain `cmake` for build and install. :confused: 

P.S.: Maybe you would like to define a few more variables, like [`%if "%{?_lib}" == "lib64" ...`](https://git.sailfishos.org/mer-core/cmake/blob/master/rpm/macros.cmake#L20) but maybe not. Let's see what will say @neochapay about 64bit systems. It seems that Mer doesn't use suffixes.

Reply:
Honestly, I don't personally care about the Mer/Sailfish packaging, I just did the changes because @neochapay asked for it :wink:  I'll replace the hardcoded make commands, but I won't bother with the macros, someone else may look into that.